css/details.css

```diff
@@ -12,13 +24,143 @@ th {
 }
 
 td {
-	text-align: right;
+	text-align: left;
```

Comment:
text-align: center should probably be set more specifically to the input, error and result sections only.